Recruitment Process Checklist (#73)
atolye15/checklist: checklists/recruitment-process.md

---
title: Recruitment Process
category: Human Resources
date: "2020-03-11"
tags: ['recruitment', 'search', 'job-posting', 'interview', 'offer']
description: Follow the steps to find the right candidate. It helps to understand the candidate's technical and profile features  
---

- [ ] Determine the correct job position  
Define the targeted position based on the ongoing projects and growth of the team.

> - [ ] Search a candidate pool and create job posting  

Reviewer: ```suggestion
- [ ] Build a candidate pool and create job posting  
```

> Create a candidate pool by using dedicated tools and platform like Linkedin. Create the most suitable job post in order to reach to best match.

Reviewer 2: ```suggestion
Create a candidate pool by using dedicated tools and platform like LinkedIn. Create the most suitable job post in order to reach to best match.
```

---
title: Recruitment Process
category: Human Resources
date: "2020-03-11"

> tags: ['recruitment', 'search', 'job-posting', 'interview', 'offer']

Reviewer: ```suggestion
tags: ['recruitment',  'job-posting', 'interview', 'offer']
```

> description: Follow the steps to find the right candidate. It helps to understand the candidate's technical and profile features  

Reviewer: ```suggestion
description: Follow these steps to find the right candidate. It helps to understand the candidate's technical capabilities and their profile  
```

> description: Follow the steps to find the right candidate. It helps us understand the candidate's technical capabilities and profile features  

Reviewer: ```suggestion
description: Follow these steps to find the right candidate. It helps to understand the candidate's technical capabilities and their profile  
```

---

- [ ] Determine the correct job position  
Define the targeted position based on the ongoing projects and growth of the team.

- [ ] Build a candidate pool and create job posting  
Create a candidate pool by using dedicated tools and platform like LinkedIn. Create the most suitable job post in order to reach to best match.

- [ ] Make a phone call with potential candidate(s)  
Contact the potential candidate in order to get to know them better & provide them with necessary information about the team and job.

- [ ] Send an "Introduction Mail" about the interviewed position to the candidate  
Inform the candidate about related position's needs and learn more about their past experiences.

- [ ] Forward test case to the candidate  
Evaluate to candidates who respond to the introduction mail. Ask to complete a small test run in order to observe candidates technical competence. 

- [ ] Share project case work with candidate         
Ask for more extensive project from the candidate who completes test process successfully. Get the information about the level of expertise.

- [ ] Organize a job interview with the candidate   

> Organize face-to-face interview with Hiring Managers in order share more detailed information about the candidate.

Reviewer: ```suggestion
Organize a face-to-face interview with Hiring Managers in order to get more detailed information about the candidate.
```

- [ ] Final interview    

> As the last step of the recruitment process, the final interview is held with the participation of the related managers. Make an offer to the suitable candidate.

Reviewer: ```suggestion
As the last step of the recruitment process, have a final interview with the candidate and the participation related managers.
```

> As the last step of the recruitment process, we set the the final interview with the participation managers and make an offer to the suitable candidate.

Reviewer: ```suggestion
As the last step of the recruitment process, have a final interview with the candidate and the participation related managers. Make a job offer if the candidate has been found suitable.
```
